app/src/pages/api/business/resolvers/queries/getBusinessCampaigns.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryResolvers } from "api/codegen/resolvers-types";
+import { ResolversContext } from "api/graphql";
+import { getPageContentByBusinessCampaignSlug } from "lib/wordpress/getPageContentBySlug";
+
+const getBusinessCampaigns: QueryResolvers["getBusinessCampaigns"] = async (_, { database }: ResolversContext) => {
+  try {
+    const data = await database.business.businessFundingCampaignPlanModel.findAll();
```

Comment:
We are missing a filter here, to `findAll` of an active campaign, meaning that it is within the `expired_at` range.